js/modules/k6/http/http.go

```diff
@@ -240,6 +257,60 @@ func (*HTTP) request(ctx context.Context, rt *goja.Runtime, state *common.State,
 		tags["status"] = strconv.Itoa(resp.Status)
 		tags["proto"] = resp.Proto
 
+		if res.TLS != nil {
+			tlsState := res.TLS
```

Comment:
You can shorten this down to `if tlsState := res.TLS; tlsState != nil {`.

Reply:
👍 typical Go noob mistake 😄